src/wizard/wizard-dialog.ts

```diff
@@ -597,6 +708,47 @@ export class ESPHomeWizardDialog extends LitElement {
     }, 0);
   }
 
+  private async _handleEmptyConfigSubmit() {
+    const nameInput = this._inputName;
+
+    const nameValid = nameInput.reportValidity();
+    if (!nameValid) {
+      nameInput.focus();
+      return;
+    }
+
+    this._busy = true;
+    this._data.type = "empty";
+    this._data.name = nameInput.value;
+
+    try {
+      const response = await createConfiguration(
+        this._data as CreateEmptyConfigParams,
+      );
+      this._configFilename = response.configuration;
+      refreshDevices();
+      this._dialog.close();
+    } catch (err: any) {
+      this._error = err.message || err;
+    } finally {
+      this._busy = false;
+    }
+  }
+
+  private async _handleConfigFileUpload() {
+    const input = this._configFileInput;
+    const file = input.files?.[0];
+    const response = await createConfiguration({
+      type: "upload",
+      name: file?.name.replace(/\.ya?ml$/, ""),
+      file_content: btoa(file ? await file.text() : ""),
```

Comment:
Looks like btoa doesnt work on utf-8 files and just expects binary, which means the unicode symbols get mangled.